code/modules/mob/living/carbon/xenomorph/strains/castes/drone/gardener.dm

```diff
@@ -368,9 +364,6 @@
 
 	var/mutable_appearance/fruit_sac_overlay_icon
 
-/datum/behavior_delegate/drone_gardener/add_to_xeno()
-	on_update_icons()
-
```

Comment:
Does this not need to be called somewhere?

Reply:
It's done in `/datum/xeno_strain/proc/_add_to_xeno()` just after the behavior delegate is added:
https://github.com/cmss13-devs/cmss13/blob/e129cfd6494555b5c8b44d529cab7c5258a454d4/code/modules/mob/living/carbon/xenomorph/strains/xeno_strain.dm#L42-L46
`/mob/living/carbon/xenomorph/update_icons()` calls `on_update_icons()` on the behaviour delegate, so it has the same end effect.

(Probably should have mentioned this in the commit or something when I removed `/datum/behavior_delegate/drone_gardener/add_to_xeno()`, since it is a bit obscured)